wdio.browserstack.config.js

```diff
@@ -0,0 +1,62 @@
+const config = {
+  hostname: 'hub.browserstack.com',
+  port: 443,
+  baseUrl: '',
+  user: process.env.BROWSERSTACK_USER,
```

Comment:
Not sure yet if these E2E tests are also meant to be run on our local machines. If so, we can consider adding [`dotenv`](https://github.com/motdotla/dotenv) to our stack.

Reply:
When the e2e tests fail on CI server, we'll probably want to try it again on our local machines. So let's consider adding it :)